app/models/user.rb

```diff
@@ -435,10 +435,6 @@ def org_admin?(organization)
 
   def block; end
 
-  def all_blocking
```

Comment:
I essentially moved this to _one place_ on the `UserBlock` model, to reduce what the `User` cares about.